src/librustdoc/doctest.rs

```diff
@@ -531,7 +531,7 @@ crate fn make_test(
         if let Some(cratename) = cratename {
             // Make sure its actually used if not included.
```

Comment:
I know this was pre-existing, but while you're at it do you want to fix this?

```suggestion
            // Make sure it's actually used if not included.
```

Reply:
Also, I'm not quite sure what this comment means.

Reply:
Does https://github.com/rust-lang/rust/pull/79775/commits/a7c197d773dd18affb2443e0ace5c2b9c7b0a44c help?

Reply:
Yes, thanks!